Library/DataSource/UITableView-Extensions.swift

```diff
@@ -1,18 +1,42 @@
 import UIKit
 
 public extension UITableView {
-  public func registerCellClass <CellClass: UITableViewCell> (_ cellClass: CellClass.Type) {
-    register(cellClass, forCellReuseIdentifier: cellClass.description())
+  public func registerHeaderFooterClass(_ headerFooterClass: UITableViewHeaderFooterView.Type) {
+    let className = self.classNameWithoutModule(headerFooterClass)
+    self.register(headerFooterClass, forHeaderFooterViewReuseIdentifier: className)
   }
 
+  public func registerCellClass(_ cellClass: UITableViewCell.Type) {
```

Comment:
Why the change?

Reply:
Making this more consistent with the rest of these functions.